### PR TITLE
Updated Nancy from 1.4.1 to 1.4.3

### PR DESCRIPTION
**Pull request by simo9000**
### PR checklist

- [X] Read the [contribution guildelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [X] Ran the shell/batch script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates)
- [X] Filed the PR against the correct branch: master for non-breaking changes and `2.3.0` branch for breaking (non-backward compatible) changes.

### Description of the PR

Updating the Nancy reference to version 1.4.3 in the project and packages config mustache files

### CODE REVIEW

File: modules/swagger-codegen/src/main/resources/nancyfx/Project.mustache [hunk 1]
```diff
@@ -39,8 +39,8 @@
         <DocumentationFile>bin\Release\{{packageName}}.XML</DocumentationFile>
     </PropertyGroup>
     <ItemGroup>
-        <Reference Include="Nancy, Version=1.4.1.0, Culture=neutral, processorArchitecture=MSIL">
-            <HintPath>..\..\packages\Nancy.1.4.1\lib\net40\Nancy.dll</HintPath>
+        <Reference Include="Nancy, Version=1.4.2.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (jimschubert) [inline modules/swagger-codegen/src/main/resources/nancyfx/Project.mustache]: Is 1.4.2 on this line correct? Or is this a typo?

Comment (simo9000) [inline modules/swagger-codegen/src/main/resources/nancyfx/Project.mustache]: I know it's weird, but I actually did a nuget update-package on the petstore example and this is exactly how it modified the project file. It compiled so i didn't ask any questions.

Comment (jimschubert) [inline modules/swagger-codegen/src/main/resources/nancyfx/Project.mustache]: That is weird. Nuget does that sometimes, I just wanted to check that it wasn't a typo. I think this version is the version from nuspec while the following line is from assembly info.

Comment (simo9000) [inline modules/swagger-codegen/src/main/resources/nancyfx/Project.mustache]: yeah and it's a major pain to keep the two consistent with very little benefit.

**Comment by wing328** [comment]
@simo9000 thanks for the PR, which has been merged into master.

**Pull request merged**